lib/index.js

```diff
@@ -101,6 +101,11 @@ const hydrate = (fields, doc, type) => {
     }
   }
 
+  const root = dotProp.get(doc, '');
```

Comment:
@joeosburn what's the purpose of this line?

Reply:
What are you getting at exactly? I'm just trying to not call `dotProp.get(doc, '');` twice

Reply:
We can discuss tomorrow.